_posts/2020-06-09-moveit2-robotic-application.md

```diff
@@ -0,0 +1,26 @@
+---
+author: Dr. John Doe, Controls Engineer - Jorge Nicho, ROS-Industrial
```

Comment:
```suggestion
author: Jorge Nicho, ROS-Industrial
```
@JafarAbdi [John Doe](https://en.wikipedia.org/wiki/John_Doe) is a dummy name in American English, and not suppose to be filled in here ;-)